src/assets/js/sfu/room.js

```diff
@@ -56,10 +57,11 @@ export default class Room extends EventEmitter {
         return audioProducer;
     }
 
-    async sendVideo(track) {
+    async sendVideo(track, data) {
         console.warn("room.sendVideo()");
         const videoProducer = await this.sendTransport.produce({
             track: track,
+            appData: data,
```

Comment:
Apparently we can sent some appData what is great maybe if we have a moderator at one point we can show that in the video-widget